BackofficeBundle/Controller/InternalLinkController.php

```diff
@@ -18,13 +18,21 @@ class InternalLinkController extends AbstractAdminController
      */
     public function formAction(Request $request)
     {
+        $option = array('action' => $this->generateUrl('open_orchestra_backoffice_internal_link_form'));
+
+        $option["method"] = "POST";
+        if ("PATCH" === $request->getMethod()) {
+            $option["csrf_protection"] = false;
```

Comment:
In InternalLinkFormView.coffee, line 46, I inject with PATCH method data from data-options to reload the internal link popin and allow edition. This will result in csrf error, if I keep csrf_protection.